docs/89-NEWS.md

```diff
@@ -1,10 +1,186 @@
 # NEWS
 
-<a name="v42"></a>
-# Awesome window manager framework version 4.2 changes
+<a name="v43"></a>
+# Awesome window manager framework version 4.3 changes
 
 <center> <img src="../images/AUTOGEN_wibox_logo_logo_and_name.svg" /> </center>
 
+Awesome v4.3 is the third release of the 4.x API. It comes after a year of
+little fixes and improvements. Awesome v4.2 was very stable thanks to everybody
+effort to unit test everything. Given no major bug warranted a new release,
+this one adds a few large features while preserving full compatibility with
+existing user configurations.
+
+## New features
+
+ * `gears.string` now has a `endswith` and `startswith` functions
+ * `luarocks` modules are now automatically available in Awesome
+ * A generic way to create or use widgets have been added
+   (`wibox.widget.base.make_widget_from_value`)
+ * It is now possible to connect to signals from all instances of a widget at once
+ * The calendar widget now supports margins
+ * The documentation has a new theme
+ * Wiboxes now have a `to_widget()` and `save_to_svg()` methods.
```

Comment:
```suggestion
 * Wiboxes now have a `to_widget()` and `save_to_svg()` method.
```

Reply:
Keeping in line with the above gears.string grammar, would we want to keep `methods` and remove the `a` instead?